packages/graphql-api/src/modules/accounts-password/resolvers/mutation.ts

```diff
@@ -16,6 +16,7 @@ export const Mutation: MutationResolvers<ModuleContext<AccountsModuleContext>> =
   },
   createUser: async (_, { user }, { injector }) => {
     const userId = await injector.get(AccountsPassword).createUser(user as PasswordCreateUserType);
+    user.email && (await injector.get(AccountsPassword).sendVerificationEmail(user.email));
```

Comment:
@sakulstra I can see a few problems of doing this here:
- we will need to apply the change to every storage (eg: REST).
- if you call the function directly from your server, this won't be triggered.

I guess the best option is to add this on the server under a flag.

Reply:
Thanks for the feedback!

> add this on the server
wasn't sure if you meant accounts-server of the server part of accounts password? I guessed accounts-password as email validation might not make a lot of sense for other authentication services?

please rereview if you got some time :)